backend/dataall/modules/shares_base/tasks/share_manager_task.py

```diff
@@ -7,10 +7,11 @@
 from dataall.base.loader import load_modules, ImportMode
 
 root = logging.getLogger()
+root.setLevel(logging.INFO)
 if not root.hasHandlers():
     root.addHandler(logging.StreamHandler(sys.stdout))
 log = logging.getLogger(__name__)
-log.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+# log.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
```

Comment:
I commented this and added this line - root.setLevel(logging.INFO) ( which was previously present ) since with the new changes the logger was not working. I think this should be fixed. I will revert back the changes . This is the GH issue were I have raise the issue with logging - https://github.com/data-dot-all/dataall/issues/1680